src/main/kotlin/process/pitch/UtauMode2PitchConversion.kt

```diff
@@ -65,7 +65,7 @@ fun pitchFromUtauTrack(pitchData: UtauTrackPitchData?, notes: List<Note>): Pitch
         lastNote = note
     }
     pitchPoints.addAll(pendingPitchPoints)
-    return Pitch(pitchPoints, isAbsolute = false)
+    return Pitch(pitchPoints, false)
```

Comment:
I added this because it makes it more readable (you don't need to jump to definition of this function to see what this `false` means)

Reply:
Got it! I add it back in 9b2f2c1 .